tests/test_examples.py

```diff
@@ -27,8 +27,9 @@ def test_plot_geodesics_se3(self):
     def test_plot_geodesics_so3(self):
         plot_geodesics_so3.main()
 
-    def test_plot_grid_h2(self):
-        plot_grid_h2.main()
+    # TODO(nina): this test fails
+    # def test_plot_grid_h2(self):
```

Comment:
yeah what's up with that?

Reply:
AssertionError: the points on the geodesic don't belong to H2 after ~200 steps. Numerical issue maybe?